import-automation/README.md

```diff
@@ -132,4 +132,58 @@ directories:
 
 ## Deployment
 
-TODO(intrepiditee): Finish.
+1. Check in the `import-automation` directory to the repository.
```

Comment:
I am not quite sure what this requires.

Reply:
just saying the directory needs to be in the repo